app/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,13 @@ integration enabled on your home assistant instance.</string>
   <string name="error_onboarding_connection_failed">Unable to connect to Home Assistant.</string>
   <string name="error_connection_failed">Unable to connect to Home Assistant.</string>
   <string name="webview_error">There was an error loading Home Assistant, please review the connection settings and try again.</string>
+  <string name="webview_error_description">Encountered error :</string>
+  <string name="webview_error_SSL_DATE_INVALID">The date of the Home Assistant certificate is invalid, please review the Home Assistant certificate or the connection settings and try again.</string>
+  <string name="webview_error_SSL_EXPIRED">The Home Assistant certificate has expired, please review the Home Assistant certificate or the connection settings and try again.</string>
```

Comment:
Should we consider making this more informative? Such as please renew the certificate and don't forget to restart HA for the change to take place.  Maybe even drop a hint to use the `cert_expiry` sensor to prevent this from happening.  I think the more helpful we are here the less frustrated users will be.